apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -94,17 +94,24 @@ class VotePanelContent extends React.Component {
   }
   loadUserCanVote = (user, vote) => {
     const { app } = this.props
-    if (user && vote) {
-      this.setState({ loadingCanVote: true })
-
-      // Get if user can vote
-      app
-        .call('canVote', vote.voteId, user)
-        .first()
-        .subscribe(canVote => {
-          this.setState({ loadingCanVote: false, userCanVote: canVote })
-        })
+    // Note: the next section is commented until we get the connect button in
+    // the MenuPanel. Until then, the user need to access the signing panel
+    // in order to connect, so we consider that the user can vote.
+    if (vote) {
```

Comment:
I had a few thoughts on this, especially in the context of responsive screens: https://github.com/aragon/aragon/issues/315#issuecomment-461778318.

I think we can allow the user to vote if their account isn't connected, but still check that they can actually vote if an account is available.